### PR TITLE
Update CreateAnExperiment.md with latest guideline

### PR DESCRIPTION
**Pull request by safern**

### CODE REVIEW

File: CreateAnExperiment.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ Experiments should be contained within a branch in the dotnet/runtimelab reposit
 
 - Pick a good name for your experiment and create branch for it in dotnet/runtimelab. Branch names should be prefixed with `feature/` in order to have official build support.
    - If the experiment is expected to require changes of .NET runtime itself, it should be branched off of [dotnet/runtimelab:runtime-master](https://github.com/dotnet/runtimelab/tree/runtime-master) that is a manually maitained mirror of [dotnet/runtime:master](https://github.com/dotnet/runtime/tree/master).
```
Comment (jkotas) [inline CreateAnExperiment.md:8]:
```suggestion
   - If the experiment is expected to require changes of .NET runtime itself, it should be branched off of [dotnet/runtimelab:runtime-master](https://github.com/dotnet/runtimelab/tree/runtime-master) that is a automatically maitained mirror of [dotnet/runtime:master](https://github.com/dotnet/runtime/tree/master).
```

**Review by jkotas — APPROVED** [review]

File: CreateAnExperiment.md [hunk 2]
```diff
@@ -5,11 +5,13 @@ Experiments should be contained within a branch in the dotnet/runtimelab reposit
...
 - Create label `area-<your experiment name>` for tagging issues. The label should use color `#d4c5f9`. 
 - Edit `README.MD` in your experiment branch to include details about the experiment. Example: [README.md](https://github.com/dotnet/runtimelab/blob/feature/NativeAOT/README.md).
 - If your experiment is branched from dotnet/runtime:
+   - Update the pre-release label to include a unique identifier representing the name of the experiment to avoid package clashes given that all experiments publish to the same [feed](https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet-experimental). To do this you need to update the versioning properties in [`Versions.props`](https://github.com/dotnet/runtimelab/blob/0cf87055346fd12fb22478f17521ebeb28a6d323/eng/Versions.props#L9)
    - Enable CI builds by editing `eng/pipelines/runtimelab.yml` in your branch. Example: [#137](https://github.com/dotnet/runtimelab/pull/137/files)
```
Comment (safern) [inline CreateAnExperiment.md:15]:
```suggestion
   - Edit `eng/pipelines/runtimelab.yml` in your branch to just build what your experiment needs on CI.
```

**Comment by akoeplinger** [comment]
@jkotas @safern I don't see a `standalone-experiment` branch in the repo, did you mean `standalone-template`?

**Comment by safern** [comment]
Arghhh yeah...https://github.com/dotnet/runtimelab/pull/299

**Pull request merged**